JSDemos/Demos/Charts/Strips/description.md

```diff
@@ -0,0 +1,5 @@
+A strip is a colored piece of the **Chart**'s background that highlights a range of values.
+
+To configure strips, declare an array of objects in the **argumentAxis**.[strips[]](Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/strips/) or **valueAxis**.[strips[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/) option. Each object in this array configures and individual strip. This demo shows two strips on the value axis, and how to configure the [label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/)'s [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/#text) and **font**.[color](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/font/#color) options for each strip separately.
```

Comment:
> ... configures and individual strip. This demo shows two strips ...

There's gotta be information about startValue and endValue somewhere in between these two sentences.

> This demo shows **two strips** on the value axis, and **how to configure** ...

This sentence reads poorly because "and" connects different parts of speech. I suggest splitting it in two.